### PR TITLE
3search page

### PR DESCRIPTION
**Pull request by trichoplax**

### CODE REVIEW

File: public/css/style.css [hunk 1]
```diff
@@ -17,6 +17,278 @@ body {
...
+    top: 30px;
+    left: 50%;
+    margin-left: -164px;
+}
+
+.logo img {
+    width: 80px;
+    margin-left: 124px;
+    opacity: .9;
+    box-shadow: 0 3px 7px rgba(0, 0, 0, 0.25), 0 2px 2px rgba(0, 0, 0, 0.22);
```
Comment (jturner11) [inline public/css/style.css]: Needs to be hex

File: public/css/style.css [hunk 2]
```diff
@@ -17,6 +17,278 @@ body {
...
+    border-radius: 50%;
+}
+
+.logo h1 {
+    color: #ffffff;
+    text-transform: uppercase;
+    text-align: center;
+    font-size: 2rem;
+    letter-spacing: 2px;
+    text-shadow: 0 3px 7px rgba(0, 0, 0, 0.25), 0 2px 2px rgba(0, 0, 0, 0.22);
```
Comment (jturner11) [inline public/css/style.css]: Needs to be hex

File: templates/index.phtml [hunk 3]
```diff
@@ -13,6 +13,30 @@
...
+            <img src="./img/logo_4.png" alt="Awesome Badger logo">
+        </a>
+        <h1>Awesome badger bookings</h1>
+    </div>
+    <div class="date_range_picker">
+        <form action="#">
+            <div class="form_content">
+                <input type="text" id="from" name="from" placeholder="From...">
+                <input type="text" id="to" name="to" placeholder="To...">
+                <input type="submit" class="date_submit">
```
Comment (jturner11) [inline templates/index.phtml]: need to change the date so there is a minimum stay.

**Review by jturner11 — CHANGES_REQUESTED** [review]

File: templates/index.phtml (hunk 3 above)
Comment (jturner11) [reply templates/index.phtml]: please use the word 'search'

File: templates/index.phtml [hunk 4]
```diff
@@ -13,6 +13,30 @@
...
+            </div>
+
```
Comment (jturner11) [inline templates/index.phtml]: remove space please

File: templates/index.phtml [hunk 5]
```diff
@@ -13,6 +13,30 @@
...
+        </form>
+    </div>
+    <div class="scroll">
+        <p>Checkout</p>
```
Comment (jturner11) [inline templates/index.phtml]: change to 'Take a look

**Review by jturner11 — CHANGES_REQUESTED** [review]

File: templates/index.phtml (hunk 3 above)
Comment (AlexK89) [reply templates/index.phtml]: fixed in 3resultpage

File: templates/index.phtml (hunk 5 above)
Comment (AlexK89) [reply templates/index.phtml]: fixed in 3resultpage

**Pull request merged**